embulk-input-jdbc/src/main/java/org/embulk/input/jdbc/JdbcInputConnection.java

```diff
@@ -146,25 +152,90 @@ protected String quoteIdentifierString(String str)
 
     protected String buildTableName(String tableName)
     {
-    	return quoteIdentifierString(tableName);
+        return quoteIdentifierString(tableName);
     }
 
     public String buildSelectQuery(String tableName,
             Optional<String> selectColumnList, Optional<String> whereCondition,
-            Optional<String> orderByColumn)
+            Optional<String> orderByColumn) throws SQLException
```

Comment:
orderByColumn is not a column name. It is an expression including DESC or ASC options.